templates/base.html

```diff
@@ -76,11 +76,13 @@
   <!-- Page-Specific Head -->
   {% block extrahead %}{% endblock extrahead %}
 
+  <script src="{{ get_asset('spam_referrals_blocker') }}"></script>
   {% if config.google_analytics_id %}
-    <script src="https://github.com/MohamedBassem/spam-referrals-blocker/releases/download/v0.2/spam-referrals-blocker.min.js"></script>
     <!-- Google Analytics -->
     <script>
-      if(!isSpamReferral()){
+      if(!window.isSpamReferral() &&
+          window.location.host !== 'oddsite.hexxie.com:3000' &&
+          window.location.host !== 'localhost:3000') {
```

Comment:
Can we just do a `window.isDevelopment()` that we can add, say, `127.0.0.1` to, etc?